supervisor/utils/blockbuster.py

```
@@ -1,24 +1,62 @@
 """Activate and deactivate blockbuster for finding blocking I/O."""
 
-from functools import cache
+import logging
+
+from blockbuster import BlockBuster as _BlockBuster
+
+_LOGGER: logging.Logger = logging.getLogger(__name__)
+
+
+class BlockBuster:
```

Ruff (0.11.9)

Missing docstring in public class

(D101)